tests/testutils/otlp_receiver_sink.go

```diff
@@ -125,14 +137,33 @@ func (otlp *OTLPReceiverSink) Start() error {
 	if err := otlp.assertBuilt("Start"); err != nil {
 		return err
 	}
-	return (*otlp.metricsReceiver).Start(context.Background(), otlp.Host)
+
+	if err := (*otlp.metricsReceiver).Start(context.Background(), otlp.Host); err != nil {
+		return err
+	}
+
+	if err := (*otlp.tracesReceiver).Start(context.Background(), otlp.Host); err != nil {
+		_ = (*otlp.metricsReceiver).Shutdown(context.Background())
```

Comment:
I think the otlp receiver uses singletons so as long as the factory methods are called everything is hooked up w/ starting an individual, which is why there isn't an existing logs receiver start call.

Reply:
I wasn't aware of that, let me remove it and test it locally.